Sources/ArcGISToolkit/Components/JobManager/JobManager.swift

```diff
@@ -107,20 +111,37 @@ public class JobManager: ObservableObject {
     /// The operating system ultimately decides when to allow a background task to run.
     /// If you enable background status checks then you must also make sure to have enabled
     /// the "Background fetch" background mode in your application settings.
-    /// - Note: You must also add "com.esri.ArcGISToolkit.jobManager.statusCheck" to the "Permitted
-    /// background task scheduler identifiers" in your application's plist file. This only works on
-    /// device and not on the simulator.
+    ///
+    /// - Note: You must also add the ``statusChecksTaskIdentifier`` to the "Permitted
+    /// background task scheduler identifiers" in your application's plist file.
+    /// The status checks task identifier will be "com.esri.ArcGISToolkit.jobManager.statusCheck" if using the shared instance.
+    /// If you are using a job manager instance that you created with a specific ID, then the
+    /// identifier will be "com.esri.ArcGISToolkit.jobManager.<id>.statusCheck".
+    ///
+    /// Background checks only work on device and not on the simulator.
     /// More information can be found [here](https://developer.apple.com/documentation/backgroundtasks/refreshing_and_maintaining_your_app_using_background_tasks).
     public var preferredBackgroundStatusCheckSchedule: BackgroundStatusCheckSchedule = .disabled
     
     /// The background task identifier for status checks.
-    private let statusChecksTaskIdentifier = "com.esri.ArcGISToolkit.jobManager.statusCheck"
+    /// - SeeAlso ``preferredBackgroundStatusCheckSchedule``
+    public var statusChecksTaskIdentifier: String {
+        if let id {
+            "com.esri.ArcGISToolkit.jobManager.\(id).statusCheck"
+        } else {
+            "com.esri.ArcGISToolkit.jobManager.statusCheck"
+        }
+    }
     
     // A Boolean value indicating whether a background status check is scheduled.
     private var isBackgroundStatusChecksScheduled = false
     
+    /// The id of the job manager. The shared instance does not have an id.
+    var id: String?
+    
     /// An initializer for the job manager.
-    private init() {
+    init(id: String?) {
```

Comment:
```suggestion
    private init(id: String?) {
```
Should this initializer be `private`, and in the unit tests use the `init(uniqueID:)` convenience initializer instead?

Reply:
I think that's a good suggestion